packages/web-components/src/components/va-file-input/va-file-input.tsx

```diff
@@ -66,7 +66,7 @@ export class VaFileInput {
   /**
    * Optionally allow multiple files (USWDS Only)
    */
-  @Prop() multiple?: boolean = false;
+  // @Prop() multiple?: boolean = false; NOTE: disabling temporarily
```

Comment:
Can we link to the ticket that is determining if this will stay or go? Is there an accessibility ticket or something that is researching it?

Reply:
I could link to the Slack message where Matt said to leave it out, but I don't think we have a ticket yet to research this specifically.

Reply:
@rsmithadhoc do you have a ticket about checking on the a11y of multiple that you can link to here?

Reply:
@Andrew565 Good idea to link to the Slack thread! Here's Matt's feedback about the situation: https://dsva.slack.com/archives/G01BJ3ESXL4/p1706883821226859?thread_ts=1706827486.794579&cid=G01BJ3ESXL4